src/transmogrify/dexterity/tests/schemaupdater.txt

```diff
@@ -59,15 +58,15 @@ Values from a dict:
     <plone.namedfile.file.NamedFile object at ...>
     >>> spam.test_file.filename
     'zptlogo.gif'
-    >>> spam.test_file.data == zptlogo_converted
+    >>> spam.test_file.data == bytes(zptlogo, "utf-8")
     True
 
 get the filename in a seperated value from the pipeline:
     >>> two.test_file
     <plone.namedfile.file.NamedFile object at ...>
     >>> two.test_file.filename
     'testlogo.gif'
-    >>> two.test_file.data == zptlogo_converted
+    >>> two.test_file.data == bytes(zptlogo, "utf-8")
```

Comment:
`zptlogo` variable itself can be byte instead of converting here.

```python
zptlogo = (
    b"GIF89a\x10\x00\x10\x00\xd5\x00\x00\xff\xff\xff\xff\xff\xfe\xfc\xfd\xfd"
    b"\xfa\xfb\xfc\xf7\xf9\xfa\xf5\xf8\xf9\xf3\xf6\xf8\xf2\xf5\xf7\xf0\xf4\xf6"
   ...
)
```

Reply:
It fails when we set the value on the schema